build-coordinator/src/test/java/org/jboss/pnc/coordinator/test/ProjectWithDependenciesBuildTest.java

```diff
@@ -55,20 +57,24 @@
     Set<BuildSetStatusChangedEvent> eventsReceived = new HashSet<>();
 
     private void collectEvent(BuildSetStatusChangedEvent buildSetStatusChangedEvent) {
+        log.info("############Got: build set statuc changed event for build id: " + BUILD_SET_ID);
```

Comment:
Remove extra markers `#`. If this message is more important than others, try to lower the "level" of others.